src/components/ui/Merge/Bro/MergeProcessingModal.tsx

```
@@ -0,0 +1,113 @@
+    isPending: isPendingApproveBroToken,
+    isConfirming: isConfirmingApproveBroToken,
+    isError: isErrorApproveBroToken,
+    approve: approveBroToken,
+  } = useApproveToken();
+
+  const {
+    hash: hashEatBroToken,
+    isPending: isPendingEatBroToken,
+    isConfirming: isConfirmingEatBroToken,
+    isError: isErrorEatBroToken,
+    eat: eat,
```

Biome

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)